library/x509_crt.c

```diff
@@ -2951,23 +2951,138 @@ static int x509_crt_verify_chain(
     }
 }
 
+static int parse_ipv4( const char* h, char *addr )
```

Comment:
Could you add a comment explaining what the return value of `parse_ipv4()` means? Hopefully it matches the same return style as `x509_crt_check_cn()`.